UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/wwwroot/js/Custom/RadioNavigationalWarnings-admin.js

```diff
@@ -31,3 +31,47 @@ function RnwFilterList(pageSelectedIndex) {
     rnwUrl = rnwUrl.replace("amp;", "");
     window.location.href = rnwUrl;
 }
+
```

Comment:
As we are splitting Admin Web App, please add separate js file for public RNW pages.

Reply:
done required changes